L1Trigger/TrackTrigger/plugins/TTStubBuilder.cc

```diff
@@ -158,10 +158,10 @@ void TTStubBuilder<Ref_Phase2TrackerDigi_>::produce(edm::Event& iEvent, const ed
 
         tempTTStub2.addClusterRef((tempTTStub.getClusterRef(0)));
         tempTTStub2.addClusterRef((tempTTStub.getClusterRef(1)));
-        tempTTStub2.setTriggerDisplacement(2. * tempTTStub.getTriggerDisplacement());
-        tempTTStub2.setTriggerOffset(2. * tempTTStub.getTriggerOffset());
-        tempTTStub2.setRealTriggerOffset(2. * tempTTStub.getRealTriggerOffset());
-        tempTTStub2.setHardwareBend(tempTTStub.getHardwareBend());
+        tempTTStub2.setRawBend(2. * tempTTStub.getRawBend());
+        tempTTStub2.setBendOffset(2. * tempTTStub.getBendOffset());
+        tempTTStub2.setBendBE(tempTTStub.getBendBE());
```

Comment:
it seems like this doesn't follow "get full units, set half units", which emphasizes my point above...